ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs

```diff
@@ -417,49 +417,59 @@ private List<StageDamageCurve> GetStageDamageCurves()
 
         #region validation
 
-        private string IsFrequencyRelationshipValid()
+        private FdaValidationResult IsFrequencyRelationshipValid()
         {
-            return SelectedFrequencyElement.ChildElement == null ? "A Frequency Relationship is required." : null;      
+            FdaValidationResult vr = new FdaValidationResult();
+            if(SelectedFrequencyElement.ChildElement == null)
+            {
+                vr.AddErrorMessage("A Frequency Relationship is required.");
+            }
+            return vr;
         }
 
-        private string IsRatingCurveValid()
+        private FdaValidationResult IsRatingCurveValid()
```

Comment:
Follow the same naming conventions as GetPlotValidationResults.  Is implies bools are returned.